replace2.js

```diff
@@ -584,20 +604,49 @@ transform.render = async (obj, divId, previousResults = {}) => {
       return `<textarea id='${elId}' ${options} style="resize:auto;"></textarea>`;
     }
 
-    // replace #YNP with Yes No input
+    // replace #YNP with Yes No input: `(1) Yes, (0) No, (99) Prefer not to answer`
     questText = questText.replace(
-      /#YNP/g, `<div class='response'><input type='radio' id="${questID}_1" name="${questID}" value="yes"></input><label for='${questID}_1'>Yes</label></div><div class='response'><input type='radio' id="${questID}_0" name="${questID}" value="no"></input><label for='${questID}_0'>No</label></div><div class='response'><input type='radio' id="${questID}_99" name="${questID}" value="prefer not to answer"></input><label for='${questID}_99'>Prefer not to answer</label></div>`
-      // `(1) Yes
-      //  (0) No
-      //  (99) Prefer not to answer`
+      /#YNP/g,
+      `<div role="radiogroup" aria-labelledby="yesNoDontKnowLabel">
+        <label id="yesNoDontKnowLabel" class="sr-only">Select "Yes," "No," or "Prefer not to answer" to answer the question.</label>
+        <ul>
+          <li class='response'>
+            <input type='radio' id="${questID}_1" name="${questID}" value="yes">
+            <label for='${questID}_1'>Yes</label>
```

Comment:
nice, literally refactored this to display better in my PR too :D

Reply:
Great